code/modules/power/supermatter/supermatter.dm

```diff
@@ -852,13 +852,17 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 
 /obj/machinery/power/supermatter_crystal/bullet_act(obj/projectile/projectile)
 	var/turf/local_turf = loc
+	var/kiss_power = istype(projectile, /obj/projectile/kiss/death) ? 20000 : istype(projectile, /obj/projectile/kiss) ? 100 : 0
```

Comment:
Why is the second ? needed

Reply:
So different kisses give SM different power.

Reply:
> So different kisses give SM different power.

What if you make it start gracefully if blown a kiss by an Engineer or the CE (more power). But if anyone else sends a kiss it also starts but at 99% integrity?